src/ansys/grantami/bomanalytics/_item_results.py

```diff
@@ -1893,18 +1977,77 @@ def transport_reference(self) -> TransportReference:
         """
         return self._transport_reference
 
-    @property
-    def distance(self) -> ValueWithUnit:
-        """Distance travelled in the transport stage."""
-        return self._distance
-
     def __repr__(self) -> str:
         return (
             f"<{self.__class__.__name__}('{self.name}',"
             f" EE%={self.embodied_energy_percentage}, CC%={self.climate_change_percentage})>"
         )
 
 
+class TransportSummaryByCategoryResult(TransportSummaryBase):
+    """
+    Aggregated sustainability summary for all transport stages in a category.
+
+    .. versionadded:: 2.3
+    """
+
+    def __init__(
+        self,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(**kwargs)
+
+
+class TransportSummaryByPartResult(TransportSummaryBase):
+    """
+    Aggregated sustainability summary for all transport stages associated with a part.
+
+    .. versionadded:: 2.3
+    """
+
+    def __init__(
+        self,
+        part_name: Optional[str],
+        parent_part_name: Optional[str],
+        category: TransportCategory,
+        transport_types: set[str],
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(**kwargs)
+        self._part_name = part_name if part_name else None
+        self._parent_part_name = parent_part_name if parent_part_name else None
+        self._category = category
+        self._transport_types = transport_types
+
+    @property
+    def part_name(self) -> Optional[str]:
+        """
+        Name of the part (if populated in the input BoM used in the query).
+        """
+        return self._part_name
+
+    @property
+    def parent_part_name(self) -> Optional[str]:
+        """
+        Name of the parent part (if populated in the input BoM used in the query).
+        """
+        return self._parent_part_name
+
+    @property
+    def category(self) -> TransportCategory:
+        """
+        The transport category for this summary.
+        """
+        return self._category
+
+    @property
+    def transport_types(self) -> set[str]:
```

Comment:
I thought this made more sense as a set than a list, since it contains the unique set of transport types defined in the aggregation. There's no ordering and the values are unique, so list.